### PR TITLE
Issue Attributes Frontend

### PR DESCRIPTION
**Pull request by zsbrown97**
The moment we've all been waiting for...

Reporting an issue will display all possible attributes for that issue type, and viewing reported issues will include all attributes submitted for that issue

### CODE REVIEW

File: frontend/src/lib/DateTimePicker.svelte [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+<!-- distributed under the License is distributed on an "AS IS" BASIS, -->
+<!-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. -->
+<!-- See the License for the specific language governing permissions and -->
+<!-- limitations under the License. -->
+
+<script>
+  import { onMount, onDestroy, createEventDispatcher } from "svelte";
+  import "flatpickr/dist/flatpickr.min.css";
+  import flatpickr from "flatpickr";
+  import issueTime from "../stores/issueTime"
```
Comment (JackDiSalvatore) [inline frontend/src/lib/DateTimePicker.svelte:19]: import `issueTime` is not used

File: frontend/src/lib/DateTimePicker.svelte [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  import resetDate from "../stores/resetDate";
+  import messages from "$media/messages.json";
+  import expandSVG from "../icons/expand.svg";
+
+  const dispatch = createEventDispatcher();
+
+  let picker;
+  // let dateTimeInput;
```
Comment (JackDiSalvatore) [inline frontend/src/lib/DateTimePicker.svelte:27]: can remove this

File: frontend/src/lib/DateTimePicker.svelte [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  let ref;
+  let value;
```
Comment (JackDiSalvatore) [inline frontend/src/lib/DateTimePicker.svelte:29]: `value` is not used

File: frontend/src/routes/+page.svelte
Comment (JackDiSalvatore) [inline frontend/src/routes/+page.svelte:1]: nitpicking: extra whitespace can be removed

**Review by JackDiSalvatore — APPROVED** [review]

**Review by HootDunk — APPROVED** [review]

**Review by JackDiSalvatore — APPROVED** [review]

**Review by JackDiSalvatore — APPROVED** [review]

**Pull request merged**